osu.Game.Tests/Visual/Online/TestSceneCommentActions.cs

```diff
@@ -204,6 +208,22 @@ public void TestDeletionFail()
             });
         }
 
+        [Test]
+        public void TestCopyLink()
+        {
+            addTestComments();
+            AddUntilStep("Comments loaded", () => this.ChildrenOfType<DrawableComment>().Count() == 2);
+            AddStep("Click copy link", () =>
+            {
+                var comment = this.ChildrenOfType<DrawableComment>().First();
+                var links = comment.ChildrenOfType<LinkFlowContainer>().Single(x => x.Name == @"Actions buttons");
+                var btn = links.ChildrenOfType<OsuSpriteText>().Single(x => x.Text.ToString().StartsWith("Copy"));
+                InputManager.MoveMouseTo(btn);
+                InputManager.Click(MouseButton.Left);
+            });
+            AddUntilStep("Link copied", () => host.GetClipboard()?.GetText() == "https://osu.ppy.sh/comments/1");
```

Comment:
This can't work because headless hosts don't get access to the clipboard (and probably for the better that they don't). Not sure this test needs to exist at all. You could

```csharp
[Ignore("Requires desktop game host for clipboard access.")]
```

if you really wanted to keep it, but an ignored test is really a dead test, so there's not much difference in principle.